pkg/ebpf/tracee.go

```diff
@@ -454,6 +464,7 @@ const (
 	optCaptureModules
 	optCgroupV1
 	optProcessInfo
+	optCaptureFDFilePath
```

Comment:
We should change "Capture" wording here. We use "capture" to "capture artifacts" and it might be a bit confusing. Anything else, no hard opinion.

Reply:
Perhaps `optTranslateFDFilePath`.